CONTRIBUTING.md

```diff
@@ -77,14 +77,19 @@ If everything looks good, they will merge the code and release a new version whi
 
 <!-- References -->
 
-[pull requests]: https://github.com/mineiros-io/terraform-google-organization-iam/pulls
-[pre-commit-file]: https://github.com/mineiros-io/terraform-google-organization-iam/blob/main/.pre-commit-config.yaml
+<!-- markdown-link-check-disable -->
```

Comment:
This isn't the latest version of the file. You may want to update the module locally :)

Reply:
forgot to commit! done